fastly/resource_fastly_service_v1.go

```diff
@@ -271,6 +271,12 @@ func resourceServiceV1() *schema.Resource {
 							Description: "SSL certificate hostname",
 							Deprecated:  "Use ssl_cert_hostname and ssl_sni_hostname instead.",
 						},
+						"ssl_ca_cert": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Default:     "",
```

Comment:
The tests pass without the default value. I put it in to match the style of similar optional parameters - e.g. `ssl_cert_hostname`.

Thanks

Tom